ntbs-service/Models/Notification.cs

```diff
@@ -60,6 +60,7 @@ public Notification()
         public virtual ComorbidityDetails ComorbidityDetails { get; set; }
         public int? GroupId { get; set; }
         public virtual NotificationGroup Group { get; set; }
+        public ICollection<Alert> Alerts { get; set; }
```

Comment:
We're currently extensively using virtual with navigation properties - it's not really required because we're not lazy loading - but think consistency is best here.